server/services/participants.js

```diff
@@ -6,6 +6,12 @@ const { dbClient, collections } = require('../db');
 const { createRows, verifyHeaders } = require('../utils');
 const { ParticipantsFinder } = require('./participants-helper');
 
+const deleteParticipant = async ({ email }) => {
```

Comment:
We might want to delete related participant statuses as well. Having orphaned `participants_status` lying around might cause unforeseen issues.